modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,535 @@
+
+In {productname} 6.2, list functionality in this regard is dependent on the state of the closest inheritable `contenteditable` attribute.
+
+If a `list` has a `contenteditable` attribute value of its own, this is, of course the closest ‘inheritable’ attribute, and the list and its child elements takes that value.
+
+If a `list` does not have a `contenteditable` attribute value of its own and the immediate parent of the `list` is a `div` which does have a `contenteditable` attribute set, the `list` will inherit the value of this attribute.
+
+However, if the `list` has a `contenteditable` attribute value of its own it will take this value rather than the value of the parent div.
+
+Likeiwse, if a child element of a `list` (for example, an `+<li>+`) has a `contenteditable` attribute value of its own, it will take this value rather than the value of any parent element.
+
+
+=== Removed extra bottom padding in the context toolbar of the TinyMCE 5 skin.
+
+In {productname} 6.1.2, extra padding was added to the context toolbar and, consequently, separator lines were incorrectly drawn on multi-line toolbars when using either the `tinymce-5` skin or premium skins.
+
+With this update the extra padding was removed and separator lines in {productname} 6.2 multi-line toolbars are drawn correctly.
+
+=== Fixed a regression where pressing Enter added or deleted content outside the selection.
+
+A regression found in {productname} 6.0 caused inline elements with `contenteditable="false"` attributes to be incorrectly removed when *Enter* or *Return* were pressed on a selection which began before, and entered into, the block containing the element.
+
+This regression was fixed in {productname} 6.2 and pressing *Enter* or *Return* in this circumstance no longer incorrectly removes these elements.
+
+=== Fixed a regression where pressing Enter deleted selected `contenteditable="false" <pre>` elements.
+
+A regression found in {productname} 6.0 caused `preformatted` elements with `contenteditable="false"` attributes to incorrectly insert a newline when *Enter* or *Return* were pressed.
+
+This regression was fixed in {productname} 6.2 and pressing *Enter* or *Return* no longer inserts an erroneous newline.
+
+
+=== The `editor.insertContent()` API did not respect the `no_events` argument.
+
+If, for example, `editor.insertContent('<p>test</p>', { no_events: true });` was run, `BeforeGetContent` and `SetContent` events were still fired, despite `no_events` being set to `true`.
+
+With this update, the API respects the `no_events: true` setting. Adding content to a {productname} document with `no_events` set to true now adds the content without firing events, as expected.
+
+[[deprecated]]
+== Deprecated
+
+=== The `autocompleter` configuration property, `ch`, has been deprecated.
+
+The `ch` autocompleter configuration property will be removed in the next major release.
+
+Use the `trigger` property instead.
```

Comment:
Should we explain why it was deprecated?

Reply:
```suggestion
The `trigger` property should be used instead, as it allows one or more characters to be specified.
```
To try and suggest something for Morgan's comment.

Reply:
(discussion): accept suggestion